iyzipay/iyzipay_resource.py

```diff
@@ -68,7 +68,10 @@ def format_header_string(api_key, hashed):
 
     @staticmethod
     def resource_pki(request):
-        return 'locale=' + request.get('locale') + ',conversationId=' + request.get('conversationId') + ','
+        if request.get('conversationId') != None:
```

Comment:
Hi,  @abdullahselek 
You should verify this implementation with unit test which sends conversationId null or empty.

Reply:
Hi @kadriyedogan, I couldn't see any unit test target for this library. Am I missing something?

Reply:
@abdullahselek Sorry, my mistake. We added unit tests for other clients such as iyzipay-java, iyzipay-ruby, iyzipay-dotnet except python client :) I see that we only have samples.
 I will create another issue to add test support.If you have any experience about unit testing and contribute to us we would be happy 👍

Reply:
@kadriyedogan no worries. When you have time please review my changes. For the new issue adding test support I'm going to make a new PR.